hasura/migrations/1676985715486_update_view_public_ZenikasAverageCurrentSkillsAndDesires/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE VIEW "public"."ZenikasAverageCurrentSkillsAndDesires" AS 
+ SELECT "Skill".id AS "skillId",
+    "Skill"."categoryId",
+    "Skill".name,
+    avg("UserSkillDesire"."skillLevel") AS "averageSkillLevel",
+    avg("UserSkillDesire"."desireLevel") AS "averageDesireLevel",
+    count("UserSkillDesire"."userEmail") AS "userCount"
```

Comment:
I think, we could add the "verified" column here

Reply:
As discussed, unverified skills are already shown in another search (on graph page). To keep consistency, we adopt this behavior for the global search.
Also, this can avoid duplicated skill during the validation period.